src/main/java/org/jabref/logic/journals/AbbreviationParser.java

```diff
@@ -81,17 +84,36 @@ private void addLine(String line) {
         if (line.startsWith("#")) {
             return;
         }
-        String[] parts = line.split("=");
-        if (parts.length == 2) {
-            final String fullName = parts[0].trim();
-            final String abbrName = parts[1].trim();
 
-            // check
+        String[] parts = line.replace(textQuote + textQuote, textQuote).split(separator);
```

Comment:
Do not implement a CSV parser by oneself.

Please use Apache CSVparser: https://commons.apache.org/proper/commons-csv/apidocs/org/apache/commons/csv/CSVParser.html

Main homepage: https://commons.apache.org/proper/commons-csv/

Use

    CSVFormat.DEFAULT
      .withIgnoreEmptyLines(true)
      .withDelimiter(';'')
      .withEscape('\\')
      .withQuote('"');

(should work - source: https://www.codota.com/code/java/methods/org.apache.commons.csv.CSVFormat/withDelimiter)


If that library does not work well, you can also use [SuperCSV](http://super-csv.github.io/super-csv/index.html).